src/serialization/ObjectData.ts

```diff
@@ -191,7 +191,14 @@ export class ObjectDataOutput implements DataOutput {
         this.writeArray(this.writeShort, shorts);
     }
 
+    /** @deprecated since version 4.1.
+     * This method will be deprecated in next major version. Please use writeString instead.
+     */
```

Comment:
Since we mention the deprecation in the interface, I don't think this documentation is necessary. Same applies to other places

Reply:
Will it be removed? Maybe I mistyped it. `This method will be removed in next major version`?

Reply:
In the next major version(5.0), we can remove it

Reply:
I deleted the `This method will be deprecated in next major version.` part. Is it fine without it too? I mean should we write to inform before removing?

Reply:
No, I guess the warning should stay until we remove it

Reply:
Ok. hopefully, I handled your points now